src/script/Config.ts

```diff
@@ -43,7 +43,8 @@ export class Configuration {
   readonly ENVIRONMENT = env.ENVIRONMENT || 'production';
   readonly FEATURE = {
     ...env.FEATURE,
-    ENABLE_EXTRA_CLIENT_ENTROPY: env.FEATURE.ENABLE_EXTRA_CLIENT_ENTROPY && Runtime.isWindows(),
+    ENABLE_EXTRA_CLIENT_ENTROPY:
+      env.FEATURE.ENABLE_EXTRA_CLIENT_ENTROPY && (this.ENVIRONMENT !== 'production' || Runtime.isWindows()),
```

Comment:
This is unrelated change (and the condition looks off)

Reply:
Talked this one through with @tlebon. 

The change happens because the windows platform should only be a blocker when the app runs in production mode. While in development, I want to see the entropy generator as soon as I enable the flag, even when on a mac (our workstations).

Reply:
gonna revert it for now

Reply:
> Maybe we could make it a faire flag for dev. This is going to be very annoying for us developers of we have to go through the process for every new device!!
> 
> Also, still this is unrelated to this PR. Shouldn't be here

I'm not clear why we would have to go thru the process for every new device? only if we have the flag enabled?

Reply:
Right, correct, there is already a flag, my mistake 😅🤦‍♂️
You can ignore my comment, gentlemen